src/scripts/appclose.ts

```diff
@@ -0,0 +1,11 @@
+((require as unknown) as typeof define)(["playbackManager"], function (playbackManager) {
```

Comment:
?

Use [@types/requirejs](https://www.npmjs.com/package/@types/requirejs) for typing require() and define().

Reply:
Require is conflicting with require from nodejs